### PR TITLE
[ISSUE #4010]fix sign plugin DataBufferLimitException error

### PR DESCRIPTION
**Pull request by ableYang123**
[BUG] <using the Sign Plugin, a DataBufferLimitException occurred when the JSON request was too large> #4010

### CODE REVIEW

File: shenyu-plugin/shenyu-plugin-sign/src/main/java/org/apache/shenyu/plugin/sign/SignPlugin.java [hunk 1]
```diff
@@ -69,6 +69,17 @@ public class SignPlugin extends AbstractShenyuPlugin {
      */
     public SignPlugin(final SignService signService) {
```
Comment (yunlongn) [inline shenyu-plugin/shenyu-plugin-sign/src/main/java/org/apache/shenyu/plugin/sign/SignPlugin.java:70]: whether to remove this constructor.

**Comment by moremind** [comment]
pls check xcu

File: shenyu-plugin/shenyu-plugin-sign/src/main/java/org/apache/shenyu/plugin/sign/SignPlugin.java (hunk 1 above)
Comment (yunlongn) [reply shenyu-plugin/shenyu-plugin-sign/src/main/java/org/apache/shenyu/plugin/sign/SignPlugin.java:70]: You can modify the unit test

Comment (ableYang123) [reply shenyu-plugin/shenyu-plugin-sign/src/main/java/org/apache/shenyu/plugin/sign/SignPlugin.java:70]: ok

**Review by yu199195 — APPROVED** [review]

**Comment by yunlongn** [comment]
@ableYang123 you can add my wechat: liyunlongn

**Pull request merged**